manifests/base/jupyter-minimal-notebook-imagestream.yaml

```diff
@@ -32,6 +33,7 @@ spec:
         opendatahub.io/notebook-software: '[{"name":"Python","version":"v3.9"}]'
         opendatahub.io/notebook-python-dependencies: '[{"name":"JupyterLab","version": "3.5"}, {"name": "Notebook","version": "6.5"}]'
         openshift.io/imported-from: quay.io/opendatahub/workbench-images
+        opendatahub.io/notebook-build-commit: "17c2e49"
       from:
         kind: DockerImage
         name: $(odh-minimal-notebook-image-n-1)
```

Comment:
Lets add the value to bottom version: `1.2`

`opendatahub.io/notebook-build-commit: 4c8f26e`
Image: `quay.io/opendatahub/notebooks:jupyter-minimal-ubi8-python-3.8-4c8f26e`